docs/howto/add_relationship_links.md

```diff
@@ -15,40 +15,42 @@ class Api::V1::UsersController < ApplicationController
       serializer: Api::V1::UserSerializer,
       include: []
   end
+end
 ```
 
 Bear in mind though that ActiveModelSerializers are [framework-agnostic](outside_controller_use.md), Rails is just a common example here.
 
 ### Links as an attribute of a resource
-**This is applicable to JSONAPI, JSON and Attributes adapters**
+**This is applicable to JSON and Attributes adapters**
 
 You can define an attribute in the resource, named `links`.
 
 ```ruby
 class Api::V1::UserSerializer < ActiveModel::Serializer
-  attributes :id, :name, :links
+  include Rails.application.routes.url_helpers
```

Comment:
> `You can define an attribute in the resource, named `links`.

Isn't this true for all the adapters? It's just especially confusing for JSONAPI. Also, the name `links` here is arbitrary once you're mixing in the url helpers and just making an attribute return an custom object (hash).

I'd also rather encourage usage of the serialization context over mixing in ` include Rails.application.routes.url_helpers` but that's me. I sorta think using the `include Rails.application.routes.url_helpers` should be a way of doing it until we unify the adapters or write better docs for serialization_context. 


Needs tests. That's part of the reason the doc fix is needed, right?  we could put that off into a followup pr?